packages/keystone/lib/adapters/index.js

```diff
@@ -25,13 +25,12 @@ class BaseKeystoneAdapter {
     // Set up all list adapters
     try {
       const taskResults = await this.postConnect();
-      const errors = taskResults.filter(({ isRejected }) => isRejected);
+      const errors = taskResults.filter(({ isRejected }) => isRejected).map(({ reason }) => reason);
 
       if (errors.length) {
         if (errors.length === 1) throw errors[0];
         const error = new Error('Multiple errors in BaseKeystoneAdapter.postConnect():');
-        error.errors = errors.map(({ reason }) => reason);
-        throw error;
+        throw { ...error, errors };
```

Comment:
Spreading like this changes it from a type of `Error` to a type of `Object` which can break other things like error checking code in the Apollo stack.

```suggestion
        error.errors = errors;
        throw error;
```